src/libs/actions/PersistedRequests.js

```diff
@@ -10,7 +10,7 @@ Onyx.connect({
 });
 
 function clear() {
```

Comment:
Can you please add JSdocs here for the `returns` that says something like:

`This promise is only used by tests. DO NOT USE THIS PROMISE IN THE APPLICATION CODE`

Reply:
I believe we are trying to avoid using that and prefer to wait for the Onyx method to finish because `waitForPromisesToResolve` is not compatible with the batched updates solution.